examples/gpu.jl

```diff
@@ -0,0 +1,28 @@
+using DFTK
+using CUDA
+
+a = 10.263141334305942  # Lattice constant in Bohr
+lattice = a / 2 .* [[0 1 1.]; [1 0 1.]; [1 1 0.]]
+Si = ElementPsp(:Si, psp=load_psp("hgh/lda/Si-q4"))
+atoms     = [Si, Si]
+positions = [ones(3)/8, -ones(3)/8]
+terms = [Kinetic(),
+            AtomicLocal(),
+            AtomicNonlocal(),
+            Ewald(),
+            PspCorrection(),
+            Entropy(),
+            Hartree()]
+# Now, build a supercell to have a larger system
+pystruct = pymatgen_structure(lattice, atoms, positions)
+pystruct.make_supercell([4,2,2])
+lattice   = load_lattice(pystruct)
+positions = load_positions(pystruct)
+atoms     = fill(Si, length(positions))
```

Comment:
We dropped pymatgen support some time ago. Just replace by doing the same thing in ASE (see https://docs.dftk.org/stable/examples/supercells/):
```suggestion
supercell = ase_atoms(lattice, atoms, positions) * (repeat, 1, 1)
lattice   = load_lattice(supercell)
positions = load_positions(supercell)
atoms     = fill(Si, length(positions))
```